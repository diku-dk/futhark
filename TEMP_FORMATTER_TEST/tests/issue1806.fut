type record = {ctx: i64, foo': i64}

entry main(r: record): record = r with ctx = 1
