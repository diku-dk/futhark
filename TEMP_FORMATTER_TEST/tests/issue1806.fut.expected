beb59c36059334ea79cafcd76386ac46
